client/src/app/pantry/pantryDisplayItem.ts

```diff
@@ -0,0 +1,9 @@
+import { Product } from '../products/product';
+
+export interface PantryDisplayItem {
+  _id: string;
+  product: Product;
+   // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
This and the next two lines are indented one space more than the lines above. We presumably want these to all be aligned.